iaac/terraform/kubeflow-components/knative-serving/locals.tf

```diff
@@ -0,0 +1,17 @@
+locals {
+  name = "knative-serving"
+
+  default_helm_config = {
```

Comment:
curious, how is having these configs separately here useful

Reply:
some of the default values are required like name, namespace
others are convenient like timeout

consolidates it in locals
```
  default_helm_config = {
    name        = local.name
    version     = "0.1.0"
    namespace   = "default"
    values      = []
    timeout     = "600"
  }

  helm_config = merge(
    local.default_helm_config,
    var.helm_config
  )
```

to pass it into main as one value
```
module "helm_addon" {
  source            = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons/helm-addon?ref=v4.9.0"
  helm_config       = local.helm_config
  addon_context     = var.addon_context
}
```

we could also consolidate it in main, i dont have a preference here